build/localnet/templates/v1-validator-configs.yaml.tpl

```diff
@@ -1,13 +1,14 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: v1-validator-default-config
```

Comment:
Instead of using template for this file, I would suggest to keep that file static, and only add an environment variable for that config parameter in `build/localnet/templates/v1-validator-template.yaml.tpl`.